src/YarnLock.php

```diff
@@ -72,17 +69,13 @@ protected static function evaluatePackages($data)
                 if (!empty($handledPackages[spl_object_id($package)])) {
```

Comment:
with PHP 8 we could use WeakMap instead of the object IDs, which seems cleaner to me